ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4733,3 +4733,13 @@ HWRWISOVPM:
   - heating_return_water_isolation_valve_percentage_sensor
   implements:
   - MONITORING
+
+CSOUC:
+  description: "Cooling occupied/unoccupied setpoint control."
+  id: "292411e6ab73486ca5bc84874ad47aaa"
+  is_abstract: true
+  uses:
+  - zone_air_unoccupied_cooling_temperature_setpoint
+  - zone_air_occupied_cooling_temperature_setpoint
+  implements:
+  - CONTROL
```

Comment:
Include the zone_air_temperature_sensor here, and change from CONTROL to OPERATIONAL. You did not include the actuator (e.g. valve or damper) that actually controls the zone to temperature.